src/pages/Home/EmulatorLogin.tsx

```diff
@@ -3,10 +3,13 @@ import { Button } from 'react-bootstrap'
 import { Link } from 'react-router-dom'
 import { TEST_ACCOUNTS } from '../../constants'
 import { signInEmulatorEmailPassword } from '../../utils/emulators'
+import { useHistory } from 'react-router-dom'
 
 const [FIRST_TEST_ACCOUNT, SECOND_TEST_ACCOUNT] = TEST_ACCOUNTS
 
 export const EmulatorLogin = ({ authInstance, token }) => {
+    const history = useHistory()
+    const isAnonymous = authInstance.currentUser?.isAnonymous
```

Comment:
we may wanna include this information in redux, dispatching it after successful login

Reply:
you're right